recipes/r-kde1d/meta.yaml

```
@@ -0,0 +1,90 @@
+{% set version = '1.0.4' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-kde1d
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/kde1d_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/kde1d/kde1d_{{ version }}.tar.gz
+  sha256: 124eb2e427e08fd24ca37e0baa697736601785d6c6fb1c97c2ebcc5146bb4556
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ compiler('cxx') }}            # [not win]
+    - {{ compiler('m2w64_cxx') }}      # [win]
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}make
+    - {{ posix }}sed               # [win]
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-bh
+    - r-rcpp
+    - r-rcppeigen
+    - r-randtoolbox
+  run:
+    - r-base
+    - {{ native }}gcc-libs         # [win]
+    - r-bh
```

```suggestion
    #- r-bh
```
`r-bh` is a compile-time dependency, so can be commented out here.